lib/rbnacl/boxes/curve25519xsalsa20poly1305.rb

```diff
@@ -64,14 +64,14 @@ module Boxes
     class Curve25519XSalsa20Poly1305
       extend Sodium
 
-      sodium_type      :box
+      sodium_type :box
       sodium_primitive :curve25519xsalsa20poly1305
-      sodium_constant  :NONCEBYTES
-      sodium_constant  :ZEROBYTES
-      sodium_constant  :BOXZEROBYTES
-      sodium_constant  :BEFORENMBYTES
-      sodium_constant  :PUBLICKEYBYTES
-      sodium_constant  :SECRETKEYBYTES, :PRIVATEKEYBYTES
+      sodium_constant :NONCEBYTES
+      sodium_constant :ZEROBYTES
+      sodium_constant :BOXZEROBYTES
+      sodium_constant :BEFORENMBYTES
+      sodium_constant :PUBLICKEYBYTES
+      sodium_constant :SECRETKEYBYTES, :PRIVATEKEYBYTES
```

Comment:
RuboCop is configurable. We can shut this particular one off. And FWIW I agree. This patch brings us close to in-line with the RuboCop defaults, so... let the bikeshedding begin! :wink: